commitlint.config.ts

```diff
@@ -23,6 +23,7 @@ function isBigBlock(line: string) {
 module.exports = {
     parserPreset: 'conventional-changelog-conventionalcommits',
     rules: {
+        'body-prose': [RuleStatus.Error, 'always'],
```

Comment:
@realmarv you put this rule the first, but strictly speaking if following alphabetical order, it should be the second; however I think here we're just following order of execution and I think it's better to place less important rules the last, so let's put this one one of the last ones, like after subject-lowercase

Reply:
Done!